docs/how-to/services-dashboard.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Services Dashboard
+description: Learn about Amplication's Services Dashboard, including service management, views, and related features.
+sidebar_label: Services Dashboard
+slug: /services-dashboard
+---
+
+The **Services Dashboard** is a central feature in Amplication that allows you to manage and oversee all the services within your projects. This dashboard provides a comprehensive view of your services, their statuses, and quick access to various management tools.
+
+## Overview
+
+The Services Dashboard offers two main views for managing your services:
+
+1. **Cards View**: The default view, presenting services as individual cards with basic information.
+2. **Grid View**: A compact, data-rich view providing detailed information about each service.
+
+You can access the Services Dashboard by clicking on the "Services" button next to the "Platform" button on the individual card associated with your project on the workspace overview page.
+
+From this dashboard, you can access all the essential parts of your service management, including sync with Git, settings, architecture view, and more.
+
+![Amplication Services Dashboard](./assets/services-dashboard/services-dashboard.png)
+
+## Managing Services
+
+From the Services Dashboard, you can access various tools and features for each service:
```

Comment:
The listed options are:
1. Not per service (so the "features for each service" is incorrect).
2. Were updated a little. Let's update it accordingly